src/script/page/AppMain.tsx

```diff
@@ -225,6 +225,7 @@ const AppMain: FC<AppMainProps> = ({
                 selfUser={selfUser}
                 isRightSidebarOpen={!!currentState}
                 openRightSidebar={toggleRightSidebar}
+                onRefresh={app.refresh}
```

Comment:
suggestion: maybe the naming is not really explicit. I feel that the app will completely reload when something (I don't know what) is refreshed. 

maybe we could try to be more explicit `onUserSupportedProtocolUpdated` (I don't really know what is the trigger here, you would probably find a better name :D )

Reply:
This one should not start with `"on"` actually, I suggest naming it `reloadApp`, it's called when user clicks an update button in a warning message we display when they do not support mls and the other user (the one we try to communicate with via 1:1 conversation) does support only mls (does not support proteus). After app is reloaded, we will re-evaluate 1:1 conversations again. There's a chance (but not guaranty), that after the reload self user's team has started supporting mls, so conversation can be finally established. 

So it's not a listener (`"onSomething"`), but rather an action (`"doSomething"`). Makes sense to you?

Reply:
Sure, maybe this makes more sense 👍